package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "altis-consent-api",
+  "private": true,
+  "description": "Javascript Consent API to read and register the current consent category.",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint:php": "vendor/squizlabs/php_codesniffer/bin/phpcs -s --standard=phpcs.ruleset.xml .",
+    "lint:js": "eslint .",
```

Comment:
Just a note here we should add an issue to add `eslint` as a dev dependency. It might not be clear you'd need it installed globally for this to work.

Reply:
I'm wondering if maybe I should just take that out. The javascript isn't strictly ours, so there may be things that end up not passing our linting unless we want to further deviate from the source.